--- a/lang/go/BUILD.bazel
+++ b/lang/go/BUILD.bazel
@@ -16,7 +16,10 @@
     importpath = "github.com/lyft/protoc-gen-star/v2/lang/go",
     importpath_aliases = ["github.com/lyft/protoc-gen-star/lang/go"],
     visibility = ["//visibility:public"],
-    deps = ["//:protoc-gen-star"],
+    deps = [
+        "//:protoc-gen-star",
+        "@org_golang_x_tools//imports",  # 调整依赖路径
+    ],
 )
 
 alias(